src/EventBus.php

```diff
@@ -99,4 +121,14 @@ public function disableCollectExceptions(): void
     {
         $this->collectExceptions = false;
     }
+
+    public function isCollectingException(): bool
```

Comment:
`isCollectingExceptions` since the bool value that this is getting is named `collectExceptions` ?

Reply:
yep, **s** is missing